rotkehlchen/inquirer.py

```diff
@@ -411,7 +461,6 @@ def _query_oracle_instances(
         `coming_from_latest_price` is used by manual latest price oracle to handle price loops.
         """
         instance = Inquirer()
```

Comment:
Why is that removed? Where did it go?

Reply:
not sure, what you pointed but we moved some common code in `_try_oracle_price_query` function

Reply:
I was pointing the line below. The cache key ... damn mobile app